Source/FactoryGame/Private/FGWorldScannableData.cpp

```diff
@@ -1,18 +1,163 @@
-// This file has been automatically generated by the Unreal Header Implementation tool
+// Copyright Coffee Stain Studios. All Rights Reserved.
 
 #include "FGWorldScannableData.h"
-#include "UObject/ObjectSaveContext.h"
+#include "EngineUtils.h"
+#include "Buildables/FGBuildableFrackingActivator.h"
+#include "FGScannableSubsystem.h"
+#include "FGItemPickup.h"
+#include "FGDropPod.h"
+#include "WorldPartition/WorldPartition.h"
+#if WITH_EDITOR
+#include "WorldPartition/WorldPartitionActorDesc.h"
+#endif
+
+FWorldScannableData::FWorldScannableData()
```

Comment:
I had to add this in order to make it build, but the latest Satisfactory update made this redundant. So this needs to be dropped.